indico/web/client/styles/partials/_buttons.scss

```diff
@@ -164,7 +164,7 @@ $i-button-spacing: 0.3rem;
 
   & ~ .i-button,
   & ~ .group {
-    margin-left: $i-button-spacing;
+    margin-right: $i-button-spacing;
```

Comment:
Are you sure this change is correct and intentional?

Before:
![image](https://github.com/indico/indico/assets/179599/a22d14bb-4f88-4813-a35a-ad92b6c3d771)

After:
![image](https://github.com/indico/indico/assets/179599/26eafb4f-c623-4e64-ad4e-04dc501c5fa4)


So it's removing spacing that was meant to be there to separate buttons/groups

Reply:
Hmmmm. Probably unintentional.

Reply:
Reverted